packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -412,6 +424,43 @@ describe('Right Click Menu integration', () => {
       // Verify element is being edited
       expect(fixture.screen.getByTestId('edit-panel-slider')).toBeDefined();
     });
+
+    it('should let a user scale and crop video', async () => {
+      const video = await addVideo();
+
+      // right click video
+      await rightClickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(video.id).node
+      );
+
+      // foreground: click 'scale and crop video' button
+      await fixture.events.click(scaleAndCropVideo());
+
+      // Verify element is being edited
+      expect(fixture.screen.getByTestId('edit-panel-slider')).toBeDefined();
+
+      // escape edit mode
+      await fixture.events.keyboard.press('Esc');
+
+      // right click video
+      await rightClickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(video.id).node
+      );
+
+      // set video as page background
+      await fixture.events.click(setAsPageBackground());
+
+      // right click video
+      await rightClickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(video.id).node
+      );
+
+      // background: click 'scale and crop video' button
+      await fixture.events.click(scaleAndCropBackgroundVideo());
+
+      // Verify element is being edited
+      expect(fixture.screen.getByTestId('edit-panel-slider')).toBeDefined();
+    });
```

Comment:
🤔 Maybe this wouldn't need to be duplicated if the karma fixture was searching for either 
- `/^Scale & Crop Background (Video|Image)/i`
- `/^Scale & Crop (Video|Image)/i`,

But if we want to keep these separate that's fine too